portfolio/src/main/webapp/style.css

```diff
@@ -73,13 +79,125 @@ li, a {
 
 /* Content */
 
+
+.main-container{
```

Comment:
There should be a space before {.

The easiest way to fix this is to format this file with `prettier --write`.